README.md

```diff
@@ -109,4 +109,4 @@ BlurhashFFi.free();
 ```
 check the [example](./example/) for more details
 
-**contributions in the form of PR's and Issues are a welcome**
\ No newline at end of file
+**contributions in the form of PR's and Issues are a welcome**
```

Comment:
@dhikshith12, i'm sorry, I see this at now. Can you revert before open a new version? 🙏 thanks